lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -52,17 +52,22 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
             rewriter.getF64FloatAttr(beta));
 
         // Expression: alpha * x + beta
-        Value alpha_x_plus_beta = rewriter.create<Torch::AtenAddScalarOp>(
-            binder.getLoc(), resultType, tensorOperand, constBeta,
-            /*alpha=*/constAlpha);
+        Value alphaMulX = rewriter.create<Torch::AtenMulScalarOp>(
+            binder.getLoc(), resultType, tensorOperand, constAlpha);
+        Value constOne = rewriter.create<Torch::ConstantFloatOp>(
```

Comment:
There is an aten minimum op that compares the result of the linear expression with a constant oneTensor a few lines below this, and for some reason it is using a constant int one instead of a constant float one. I'm sure it will get cast to a float at some point, but maybe this could just be cleaned up here to use your constOne.

If you make a change to use constOne for the oneTensor, it might also make sense just to change constantZero below to a float as well.